modules/alerts/main.tf

```diff
@@ -0,0 +1,96 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# AWS CloudWatch alarm
+# to send a notification when the alarm reaches the desired alarm state
+# ---------------------------------------------------------------------------------------------------------------------
+resource "aws_cloudwatch_metric_alarm" "ECS_Service_CPU_Usage_High" {
```

Comment:
Our coding convention requires to always use lower case for names.

Reply:
done